package.json

```diff
@@ -27,8 +27,10 @@
     "start": "node ./node_modules/react-native/local-cli/cli.js start --config ./rn-cli.config.js",
     "run-playground-ios": "cd examples/NavigationPlayground && react-native run-ios",
     "run-playground-android": "cd examples/NavigationPlayground && react-native run-android",
-    "test": "jest",
+    "test": "npm run lint && npm run flow && npm run jest",
+    "jest": "jest",
     "lint": "eslint src",
+    "format": "eslint --fix src",
```

Comment:
See my comment below :) we set it up in the way that eslint fix also runs prettier. Pluigns are set in a way that they don't conflict with each other.